lib/ip.js

```diff
@@ -1300,9 +1312,9 @@ binet.getReachability = function getReachability(src, dest) {
   const destNet = binet.getNetwork(dest);
 
   switch (destNet) {
-    case networks.IPV4:
+    case networks.INET4:
```

Comment:
Waaaaaiiiiiit... `networks.IPV4` was undefined before this PR right? So... this whole thing just never worked?

Reply:
this whole thing started because I was writing test case for getReachability in hsd xD.. so no, it never worked. It's easy to confuse, because networks.INET4 and types.IPV4 are used everywhere and unless you write test for every case or know that you already made this mistake and just review this types, it's easy to mistype.

Reply:
yeesh